doc/win32-usb-troubleshoot.md

```diff
@@ -0,0 +1,31 @@
+# Troubleshooting LIBUSB_* errors on win32 platforms
+
+nRF Connect Desktop uses [libusb](https://libusb.info/) and [node-usb](https://github.com/tessel/node-usb) to know how many USB-capable Nordic SoCs (e.g. nRF52840) are connected to your PC, and also to set them into DFU mode when they need to be re-programmed by a nRF Connect Desktop application.
+
+While this makes it easier to develop nRF Connect Desktop applications, this might cause some error to show up in some win32 environments. This document collects the known scenarios where that happens.
+
+## Windows 7 right after installation
+
+A `LIBUSB_ERROR_NOT_FOUND` error shows up when performing the following actions in this exact order:
+
+- Run the nRF Connect Desktop installer.
+  - This includes the *rules* for the right drivers, but not the actual link between a USB device and its driver.
+- Launch a nRF Connect Desktop application.
+- Plug a nRF52840 devkit into a USB port.
+- Let Windows 7 attach the right drivers.
+  - This uses the *rules* already installed for actually *binding* a driver to the newly-connected nRF USB device.
+  - A yellow bubble appears on the system tray, indicating driver installation, like this: 
+  ![screenshot](win32-drivers-installing.png)
+
+- A `LIBUSB_ERROR_NOT_FOUND` is shown on the log of the nRF Connect Desktop application.
+- After a couple of minutes, Windows 7 finishes binding the libusb driver to the nRF USB device.
+  - A yellow bubble appears from the system tray, indicating drivers are ready, like this:
```

Comment:
I don't know whether 'on the system tray' or 'from the system tray' is correct, be consistent.